src/main/kotlin/no/nav/familie/dokument/storage/encryption/EncryptedStorageConfiguration.kt

```diff
@@ -28,17 +28,21 @@ class EncryptedStorageConfiguration {
 
     @Bean(ATTACHMENT_ENCRYPTED_STORAGE)
     internal fun attachmentEncryptedStorage(@Autowired contextHolder: TokenValidationContextHolder,
-                                  @Qualifier(ATTACHMENT_GCP_STORAGE) storage: GcpStorageWrapper, encryptor: Encryptor): EncryptedStorage {
+                                            @Qualifier(ATTACHMENT_GCP_STORAGE) storage: GcpStorageWrapper,
+                                            encryptor: Encryptor): EncryptedStorage {
         return EncryptedStorage(contextHolder, storage, encryptor)
     }
 
+    @Profile("!dev")
```

Comment:
Jag fikk ikke kjørt med riktig bean lokalt hvis jeg ikke hadde denne.. Har @Primary i test config

Reply:
If you do not add  qualified names to the test configuration, maybe you do not need the @Profile annotation here.
If you use the !dev here, I am afraid the wiring can fail when the program is running with dev profile.